includes/admin/dashboard-widgets.php

```diff
@@ -175,7 +175,7 @@ function edd_load_dashboard_sales_widget( ) {
 									<?php echo get_the_title( $payment->ID ) ?>
 									&mdash; <?php echo $payment->user_info['email'] ?>
 								</a>
-								<?php if ( $payment->user_info['id'] > 0 ) {
+								<?php if ( ! empty( $payment->user_info['id'] ) && ( $payment->user_info['id'] > 0 ) ) {
```

Comment:
This fix works but let's go a step further and use `$payment->user_id` instead. That helps move us away from the `user_info` array and it's value will always be set.